### PR TITLE
Update dependencies and remove dependency on csi-powerstore

### PR DESCRIPTION
**Pull request by tdawe**
# Description
Update dependencies and remove dependency on csi-powerstore

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/128  |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Executed unit tests
- [x] E2E tests passed `ok      karavi-testing/karavi-metrics/metrics-test      417.318s`

### CODE REVIEW

**Review by shaynafinocchiaro — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -3,12 +3,13 @@ module github.com/dell/csm-metrics-powerstore
 go 1.14
 
 require (
-	github.com/dell/csi-powerstore v0.0.0-20210923102620-2b6d0e23ab2e
-	github.com/dell/gofsutil v1.6.0
+	github.com/dell/gocsi v1.3.1
 	github.com/dell/gopowerstore v1.5.0
```
Comment (tdawe) [inline go.mod:7]: This needs to be updated to the latest 1.6

Comment (tdawe) [reply go.mod]: Fixed

File: internal/common/testdata/invalid-format.yaml [hunk 2]
```diff
@@ -0,0 +1 @@
+invalid_format
```
Comment (alikdell) [inline internal/common/testdata/invalid-format.yaml:1]: Do we need new line at the end?

Comment (tdawe) [reply internal/common/testdata/invalid-format.yaml]: Added newlines here and in other files

**Review by alikdell — APPROVED** [review]

**Review by hoppea2 — APPROVED** [review]

**Pull request merged**